tests/_utils/test_directories.py

```diff
@@ -20,9 +20,9 @@
 
 class TestDirectories:
 
-    def __init__(self, cur_file=__file__) -> None:
-        self.tests_dir = os.path.dirname(os.path.dirname(cur_file))
-        self.morpheus_root = os.environ.get('MORPHEUS_ROOT', os.path.dirname(self.tests_dir))
+    def __init__(self) -> None:
+        self.morpheus_root = os.environ.get('MORPHEUS_ROOT')
```

Comment:
When `MORPHEUS_ROOT` is undefined this will cause the tests to fail in a weird way.
We should :
1) Use `os.environ['MORPHEUS_ROOT']` allowing the keyerror to be raised
2) Infer the root based upon the relative location of the file `os.environ.get('MORPHEUS_ROOT', os.path.dirname(os.path.dirname(os.path.dirname(__file__)))`

Reply:
Fixed.